cli/internal/cacheitem/create.go

```diff
@@ -0,0 +1,114 @@
+package cacheitem
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"crypto/sha512"
+	"io"
+	"os"
+	"time"
+
+	"github.com/moby/sys/sequential"
+	"github.com/vercel/turborepo/cli/internal/turbopath"
+)
+
+// Create makes a new CacheItem at the specified path.
+func Create(path turbopath.AbsoluteSystemPath) (*CacheItem, error) {
+	handle, err := os.Create(path.ToString())
```

Comment:
This should be `path.Create`, right?

Reply:
Residual because #2026 hadn't landed yet; I'll make a pass on all of these.